EvernoteOAuth.lua

```diff
@@ -200,11 +200,15 @@ function OAuth:loadPage(method, path, params, data)
   request['source'] = method == "POST" and ltn12.source.string(query) or nil
   request['sink'] = ltn12.sink.table(sink)
   request['headers'] = headers
+  self.logger(request)
 
   http.TIMEOUT, https.TIMEOUT = 10, 10
   local httpRequest = parsed.scheme == 'http' and http.request or https.request
```

Comment:
Btw, totally unrelated to this PR, but since last year http.request will take care of calling LuaSec all by itself, so all code like this can remove such complexity. See https://github.com/koreader/koreader-base/issues/594#issuecomment-484233915.